packages/stream-model-instance-handler/src/__tests__/model-instance-document-handler.test.ts

```diff
@@ -455,88 +561,10 @@ describe('ModelInstanceDocumentHandler', () => {
   })
 
-  it('Can update null genesis content to real content', async () => {
-    const genesisCommit = (await ModelInstanceDocument._makeGenesis(
-      context.api,
-      null,
-      METADATA
-    )) as SignedCommitContainer
-    await context.ipfs.dag.put(genesisCommit, FAKE_CID_1)
-
-    const payload = dagCBOR.decode(genesisCommit.linkedBlock)
-    await context.ipfs.dag.put(payload, genesisCommit.jws.link)
-
-    // apply genesis
-    const genesisCommitData = {
-      cid: FAKE_CID_1,
-      type: CommitType.GENESIS,
-      commit: payload,
-      envelope: genesisCommit.jws,
-    }
-    let state = await handler.applyCommit(genesisCommitData, context)
-
-    const state$ = TestUtils.runningState(state)
-    const doc = new ModelInstanceDocument(state$, context)
-    const signedCommit = (await doc._makeCommit(context.api, CONTENT1)) as SignedCommitContainer
-
-    await context.ipfs.dag.put(signedCommit, FAKE_CID_2)
-
-    const sPayload = dagCBOR.decode(signedCommit.linkedBlock)
-    await context.ipfs.dag.put(sPayload, signedCommit.jws.link)
-
-    // apply signed
-    const signedCommitData = {
-      cid: FAKE_CID_2,
-      type: CommitType.SIGNED,
-      commit: sPayload,
-      envelope: signedCommit.jws,
-    }
-    state = await handler.applyCommit(signedCommitData, context, state)
-    delete state.metadata.unique
-    delete state.next.metadata.unique
-    expect(state).toMatchSnapshot()
-  })
-
-  it('Can set existing content to null', async () => {
-    const genesisCommit = (await ModelInstanceDocument._makeGenesis(
-      context.api,
-      CONTENT0,
-      METADATA
-    )) as SignedCommitContainer
-    await context.ipfs.dag.put(genesisCommit, FAKE_CID_1)
-
-    const payload = dagCBOR.decode(genesisCommit.linkedBlock)
-    await context.ipfs.dag.put(payload, genesisCommit.jws.link)
-
-    // apply genesis
-    const genesisCommitData = {
-      cid: FAKE_CID_1,
-      type: CommitType.GENESIS,
-      commit: payload,
-      envelope: genesisCommit.jws,
-    }
-    let state = await handler.applyCommit(genesisCommitData, context)
-
-    const state$ = TestUtils.runningState(state)
-    const doc = new ModelInstanceDocument(state$, context)
-    const signedCommit = (await doc._makeCommit(context.api, null)) as SignedCommitContainer
-
-    await context.ipfs.dag.put(signedCommit, FAKE_CID_2)
-
-    const sPayload = dagCBOR.decode(signedCommit.linkedBlock)
-    await context.ipfs.dag.put(sPayload, signedCommit.jws.link)
-
-    // apply signed
-    const signedCommitData = {
-      cid: FAKE_CID_2,
-      type: CommitType.SIGNED,
-      commit: sPayload,
-      envelope: signedCommit.jws,
-    }
-    state = await handler.applyCommit(signedCommitData, context, state)
-    delete state.metadata.unique
-    delete state.next.metadata.unique
-    expect(state).toMatchSnapshot()
+  test('verifies the content against model schema', async () => {
+    await expect(
+      applyGenesisCommit(handler, {}, context)
```

Comment:
if the applyGenesisCommit function is only used in one test, I'd just inline it

Reply:
Would also be good to add a second test for applying a signed commit that doesn't conform to the schema, and inline and remove the applySignedCommitToValidMID function (which I believe is currently unused)